en/docs/administer/configuring-hazelcast.md

```diff
@@ -44,4 +44,22 @@ important to add all the members to the well-known members list in the `
      [clustering]
      ...
      members = ["<member-x-host>:<member-x-port>", ...]
-     ```
\ No newline at end of file
+     ```
+     
+!!! info
+    WSO2 provides Hazelcast Community Edition as its default
+    clustering engine. For clustering on a secure channel (i.e., secure
+    Hazelcast), use Hazelcast Enterprise. To integrate with Hazelcast
+    Enterprise, there are provisions to provide license key under clustering
+    configurations. Advanced users can fine-tune Hazelcast by creating a
+    `         <IS_HOME>/repository/conf/hazelcast.properties        `
```

Comment:
by creating a `hazelcast.properties` file in the `<IS_HOME>/repository/conf` directory